
This patch adds the PCI device IDs need by the ipr driver.

---


diff -puN include/linux/pci_ids.h~ipr-pci-ids include/linux/pci_ids.h
--- linux-2.6.3/include/linux/pci_ids.h~ipr-pci-ids	Fri Feb 27 11:24:58 2004
+++ linux-2.6.3-bjking1/include/linux/pci_ids.h	Fri Feb 27 11:25:05 2004
@@ -433,7 +433,9 @@
 #define PCI_DEVICE_ID_IBM_CPC710_PCI64	0x00fc
 #define PCI_DEVICE_ID_IBM_CPC710_PCI32	0x0105
 #define	PCI_DEVICE_ID_IBM_405GP		0x0156
+#define PCI_DEVICE_ID_IBM_SNIPE		0x0180
 #define PCI_DEVICE_ID_IBM_SERVERAIDI960	0x01bd
+#define PCI_DEVICE_ID_IBM_GEMSTONE		0xB166
 #define PCI_DEVICE_ID_IBM_MPIC_2	0xffff
 
 #define PCI_VENDOR_ID_COMPEX2		0x101a // pci.ids says "AT&T GIS (NCR)"

_
